Assets/Scripts/General.cs

```diff
@@ -103,8 +103,10 @@ public sealed class General : MonoBehaviour {
     public void SetPlayerToPlatform(int platformNumber = 1) {
         if (this.platforms.ContainsKey(platformNumber)) {
             this.currentPlatform = this.platforms[platformNumber];
-            this.playerController.transform.position = currentPlatform.startingPoint.transform.position;
-            this.playerController.GetComponent<Rigidbody>().velocity = this.playerController.GetComponent<Rigidbody>().angularVelocity = Vector3.zero;
+            if (this.playerController != null && this.currentPlatform != null) {
```

Comment:
I don't understand when either of there would ever be null. Sounds like a bug in themselves instead.